src/stock_server.c

```
@@ -1,34 +1,367 @@
+    queue->fds[queue->rear] = connfd;
+    queue->rear = (queue->rear + 1) % STOCK_QUEUE_SIZE;
+    V(&queue->mutex);
+    V(&queue->items);
+}
+
+/*
+ * Removes one descriptor, blocking when the bounded queue is empty.
+ */
+static int conn_queue_remove(conn_queue_t *queue)
+{
+    int connfd;
```

Clang (14.0.6)

[warning] variable 'connfd' is not initialized

(cppcoreguidelines-init-variables)